packages/slice-machine/pages/slices.tsx

```diff
@@ -145,21 +167,64 @@ const SlicesIndex = ({ env }: { env: Environment }) => {
                 </Box>
               )}
             </Box>
-          ) : null}
-          <Box>
-            {libraries &&
-              libraries.map((maybelib: LibraryState | undefined, i) => {
+          )}
+          <Box
+            sx={{
+              flex: 1,
+              display: "flex",
+              flexDirection: "column",
+            }}
+          >
+            {sliceCount == 0 ? (
+              <Box
+                sx={{
+                  flex: 1,
+                  alignItems: "center",
+                  flexDirection: "column",
+                  justifyContent: "center",
+                  display: "flex",
+                  textAlign: "center",
+                }}
+              >
+                <Heading sx={{ mb: 4 }}>Create your first Slice</Heading>
+                <Text sx={{ mb: 4 }}>
+                  Click the + button on the top right to create the first slice
+                  of your project. <br />
+                  <br /> It will be stored locally. You will then be able to
+                  push it to Prismic.
+                </Text>
+                <Button onClick={() => setIsOpen(true)}>
+                  {data.loading ? (
+                    <Spinner
+                      sx={{ position: "relative", top: "4px" }}
+                      color="#F7F7F7"
+                      size={18}
+                      mr={2}
+                    />
+                  ) : null}{" "}
```

Comment:
`{" "}` ??? Did you mean `&nbsp;` ?

Reply:
Prettier introduce this mistake I fixed it

Reply:
haha ok :) I would have called you a schlag otherwise 😂